containers/ddev-dbserver/Makefile

```
@@ -4,13 +4,9 @@ VERSION := $(shell git describe --tags --always --dirty)
 
 .PHONY: build container mariadb_containers mysql_containers multi-arch push test clean
 
-# The format here is majorversion:pinnedimageversion
-# This allows to pin the version (as currently required with 8.0.19
-MARIADB_VERSIONS_amd64=5.5:5.5 10.0:10.0 10.1:10.1 10.2:10.2 10.3:10.3 10.4:10.4 10.5:10.5
-MYSQL_VERSIONS_amd64=5.5:5.5 5.6:5.6 5.7:5.7 8.0:8.0.21
-MARIADB_VERSIONS_arm64=10.2:10.2 10.3:10.3 10.4:10.4 10.5:10.5
-# MySQL images on Docker Hub currently are amd64 only
-MYSQL_VERSIONS_arm64=
+include database-versions
+
+CURRENT_ARCH= $(shell ../get_arch.sh)
```

I'm sure it doesn't matter but I'd be happier with 
```suggestion
CURRENT_ARCH=$(shell ../get_arch.sh)
```

---

Fixed this.